ctapipe/containers.py

```diff
@@ -29,7 +29,7 @@
     "CoreParametersContainer",
     "ImageParametersContainer",
     "LeakageContainer",
-    "MonitoringCameraContainer",
+    "TelescopeMonitoringContainer",
```

Comment:
Why the change of the order of Monitoring and Telescope?

See my general comment: this change is no longer greppable.

Reply:
For class naming usually you prepend less general terms, so it may even be better to change all to the form: `DL1Container` + `TelecopeDL1Container`, rather than `DL1Container`+`DL1TelescopeContainer`.   But the current form does also make some sense, it just reads poorly

Reply:
FWIW, I also favor `DL1Container` + `TelescopeDL1Container`, then the former has the implicit `Array`-DL1Container

Reply:
We could even explicitly call it ArrayDL1Container